.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @david31982 @holgiosalos @JuanDelahozMiranda @manuelq12 @valeryibarra
```

Comment:
Please add a blank line to remove the EOF warning:
![image](https://user-images.githubusercontent.com/10209180/171203934-d9faf905-b424-499d-8182-feeb02a2f874.png)

You can setup Visual Studio Code to add this automatically:
https://stackoverflow.com/a/44704969

Reply:
Done, Thanks